### ISSUE TITLE
Feature Request: Support for Multiple IPs and Custom Login URLs in Mailu

### ISSUE THREAD

**Issue by wezbowebservices** [Mailu/Mailu#2942]
Dear Mailu Development Team,

I hope this message finds you well. I am writing to address a crucial issue that our organization has been facing while using Mailu, and to propose a feature enhancement that would greatly benefit us and, I believe, many other users.

Currently, Mailu allows us to send emails through only one IP address, which has posed a significant challenge for us. This limitation has resulted in our IP address being flagged for spam, affecting the deliverability of our emails. Additionally, it has forced us to maintain multiple backup servers, substantially increasing our infrastructure costs and complexity.

To address these issues, I kindly request the implementation of the following features:

Support for Multiple IPs: We would like to have the ability to connect Mailu to multiple IP addresses. This would not only help in load balancing but also improve our email deliverability by reducing the risk of IP blacklisting.

IP Role Configuration: For each added IP address, it would be incredibly helpful if we could define its specific role, such as being dedicated for POP3, IMAP, or both. This level of flexibility would allow us to optimize our email infrastructure according to our specific needs.

Custom Login URLs: To enhance the user experience and security, we propose the inclusion of a feature that enables custom login URLs for different domains hosted on Mailu. This would add an extra layer of branding and security for our email services.

These enhancements would not only address the current limitations of Mailu but also provide users with the flexibility and control they need to efficiently manage their email infrastructure.

I understand that implementing these features may require time and resources, but I believe they would greatly benefit the Mailu community and make Mailu an even more powerful and versatile email platform.

Thank you for your dedication to improving Mailu, and I look forward to hearing your thoughts on these proposals. If you require any further details or assistance in developing these features, please feel free to reach out.

**Comment by Diman0** [comment]
I don't think it is likely that this functionality is added soon. There is quite a large backlog of features. This is a large feature.

However, you can probably already configure postfix yourself to use multiple IP addresses.
Via [overrides[(https://mailu.io/2.0/faq.html#how-can-i-override-settings) you can override settings of postfix.

There are many tutorials on the internet for configuring postfix to send email via different IP addresses. For example:
https://shami.blog/2016/04/randomize-source-ip-addresses-with-postfix/
